_includes/contact_form.html

```diff
@@ -23,229 +18,229 @@
   let topicMap = [
     new TopicEntry(
       'Create Account',
-      "{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.creating_account }}",
+      '{{ site.data.[page.lang].settings.contact_page.support_request_form.topics.creating_account }}',
       [
         new TopicEntry(
           'Email delivery issue',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.email_delivery_issue }}",
+          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.email_delivery_issue }}',
         ),
         new TopicEntry(
           'Invalid email',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.invalid_email }}",
+          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.invalid_email }}',
         ),
         new TopicEntry(
           'Password issue',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.change_password }}",
+          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.change_password }}',
         ),
         new TopicEntry(
           'No phone',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.no_phone }}",
+          '{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.no_phone }}',
```

Comment:
It appears that the change from double quotes to single quotes created some syntax errors, because the text being injected here has apostrophes using the single quote ("I can't create my Login.gov account ...").

I think we should either:

- Revert back to double-quotes
- Or, update the text to use [smart quotes](https://smartquotesforsmartpeople.com/)
- Or, perform escaping on the inserted text so that we don't have to worry about unintentional string endings

Reply:
I chose option 3, so that we won't have to worry about whether someone uses single or double quotes in the future. I also like the idea of smart quotes, but I think it should be done as part of a sitewide audit.

Reply:
The same will need to be done for French and Spanish help topics as well.

Example:

https://github.com/18F/identity-site/blob/18188a28a9f6a2e33b04c3d2facc3456f0118ab2/_data/fr/settings.yml#L73

Reply:
Also, I was hoping we could automatically escape it here rather than in the source content, but I'm not sure if there's a Liquid filter easily available to us for this.

Reply:
Ended up writing an escape_quotes plugin, and added it to all the lines of the topicMap. That should take care of the problem for all languages and take the burden off folks adding to the settings files. Thanks for the callout!